src/librustc_mir/diagnostics.rs

```diff
@@ -1545,20 +1545,22 @@ Erroneous code example:
 ```compile_fail,E0505
 struct Value {}
```

Comment:
Is there an opinion for `struct Value {}` over `struct Value;`?

Reply:
Didn't know that an empty struct could be made like that.

I'd want to keep the `{}` so that the reader doesn't have to guess that the struct has no fields.